freqtrade/configuration/configuration.py

```diff
@@ -285,14 +281,11 @@ def _load_optimize_config(self, config: Dict[str, Any]) -> Dict[str, Any]:
         self._args_to_config(config, argname='hyperopt_min_trades',
                              logstring='Parameter --min-trades detected: {}')
 
-        return config
-
-    def _load_plot_config(self, config: Dict[str, Any]) -> Dict[str, Any]:
+    def _load_plot_config(self, config: Dict[str, Any]) -> None:
```

Comment:
I don't mind not returning `config` here - but should we really pass `config` in in that case, or should we use the `self.config` / `self._config` attribute instead (which is assigned by the resulting config object)?

The only "needed" public method is `get_config()` (`load_config()` is used but can be replaced by the above). - then we'd have it kindof aligned... ?

Reply:
Agree, will change it to `self._config`...

As for `get_config()/load_config()` -- yes, there is no actually need in using both of them publicy, but currently both are used in the freqrade codebase. Will probably change it (maybe in a sep. PR)

Reply:
No, let's postpone these changes.
Passing config to these functions used in tests and in the scripts. `load_config` is used by the scripts, so let's postpone these changes till all scripts become subcommands and this code simply disappears...